src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -215,3 +215,40 @@
     display: block;
   }
 }
+
+.queryEditor__footer {
```

Comment:
For the values below can we use the constants from oui  like `$ouiSizeM`.

Reply:
+1

Reply:
OUI's flexboxes are messed up; they simulate `gap` by adding margins on children and negative margins on parent and this occasionally brings in unpleasant side effects.